index.js

```diff
@@ -101,8 +100,15 @@ class DHT extends EventEmitter {
     if (buf.byteLength > 1) this.io.onmessage(socket, buf, rinfo)
   }
 
-  bind () {
-    return this.io.bind()
```

Comment:
Removing due not being useful for users, and it's automatically called when bootstrapping

Then we add docs for `.socket` and we fully close https://github.com/mafintosh/dht-rpc/issues/42

Reply:
dont disagree, but thats a sep PR